doc/conf.py

```diff
@@ -42,19 +42,20 @@
 on_saltstack = "SALT_ON_SALTSTACK" in os.environ
 
 project = "Salt"
-repo_primary_branch = (
-    "master"  # This is the default branch on GitHub for the Salt project
-)
-version = salt.version.__version__
-latest_release = os.environ.get(
-    "LATEST_RELEASE", "latest_release"
-)  # latest release (2019.2.3)
+# This is the default branch on GitHub for the Salt project
+repo_primary_branch = "master"
+latest_release = (
+    os.environ.get("LATEST_RELEASE")
+    or
+    # The next unreleased version
+    str(salt.version.__saltstack_version__.major)
```

Comment:
What would this end up being at various times? is the fact that we have a recent RC change this?

Reply:
This is mostly for local builds. Right now, for example, it returns `3003`. When we build, and supply `latest_release` as an env var, it takes that value instead. Before now, this was left empty and defaulting to `'latest_release'` as a textual placeholder, which Sphinx previously didn't care about, but newer versions of Sphinx throw a warning when they see that there is no release file:

```
Warning, treated as error:
<rst_prolog>:1:unknown document: /topics/releases/latest_release
make: *** [Makefile:56: html] Error 2
```